src/js/components/utility/UtilEvents.ts

```diff
@@ -48,8 +48,7 @@ export class UtilEvents {
      * @param label Data attribute to set on the element
      */
     public static triggerVueEvent(element: JQuery<HTMLElement>, name: string, label?: string): void {
-        const e = document.createEvent('HTMLEvents');
-        e.initEvent(name, true, true);
+        const e = new Event(name, { "bubbles": true, "cancelable": true });
         if (label) element.data(label, "true");
         element[0].dispatchEvent(e);
     }
```

Comment:
I haven't been able to verify that this works cause I wasn't able to find any good way of testing this, however it doesn't seem to break anything (and this way of handling events works in reloadButtonToolbar() (FormattingExtender.ts)